www/src/components/sponsors/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Pack, hierarchy } from '@visx/hierarchy';
+import { ParentSize } from '@visx/responsive';
+import { twMerge } from 'tailwind-merge';
+import { sponsors } from './data';
+
+export function Sponsors() {
+  const pack = {
+    children: sponsors,
+    name: 'root',
+    radius: 0,
+    distance: 0,
+  };
+
+  const root = React.useMemo(
+    () =>
+      hierarchy(pack)
+        .sum((d) => 1 + d?.monthlyPriceInDollars)
```

Comment:
types for `hierarchy` seem broken

verified they were also throwing errors in tanners repo.

I can `as unknown as MyType` to kill them if desired